common/app/views/fragments/analytics/google.scala.html

```diff
@@ -37,7 +37,15 @@
 
     // Please email Dotcom Platform before changing this value
     function getExperienceValue() {
-        return @if(WebpackTest.isParticipating) {'webpack'} else {'curl'};
+        @if(WebpackTest.isParticipating) {
+            return 'webpack';
+        } else {
+            @if(WebpackControl.isParticipating) {
+                return 'webpack-control';
+            } else {
+                return 'curl';
```

Comment:
Given that this is Scala generating JS, it gets a little confusing. As it stands, the above Scala will produce the following JS if the user is participating in the Webpack test:

```
function getExperienceValue() {
  return 'webpack';
}
``` 

If we updated the Scala to this:

```
@if(WebpackTest.isParticipating) {
  return 'webpack';
} else {
  @if(WebpackControl.isParticipating) {
    return 'webpack-control';
  }
}
return 'curl';
```

and the user was participating in the Webpack test, I believe the resulting JS would look like this:

```
function getExperienceValue() {
  return 'webpack';
  return 'curl';
}
``` 

Which is functionally equivalent, but slightly weird to have two return statements. Happy to update if you think this is neater